src/QuickMarcEditor/QuickMarcEditorRows/QuickMarcEditorRows.js

```diff
@@ -259,6 +269,8 @@ const QuickMarcEditorRows = ({
                           data-index={idx}
                           icon="trash"
                           onClick={showDeleteConfirmation}
+                          onFocus={focusRow}
```

Comment:
I don't think onFocus and onBlur will work as you should highlight row when modal is opened